include/net/azure_fota.rst

```diff
@@ -0,0 +1,72 @@
+.. _lib_azure_fota:
+
+Azure FOTA
+##########
+
+The Azure firmware over-the-air (Azure FOTA) library provides a way to parse `Azure IoT Hub device twin messages`_ to obtain firmware upgrade information and perform FOTA downloads.
+This library is used by the :ref:`azure_fota_sample` sample.
+
+The JSON format and content used in this library are based on the recommendations in `Azure Firmware Over The Air (FOTA)`_.
+The library processes the incoming JSON string and searches for a specific object in the device twin that contains the firmware upgrade information.
+The expected format of a ``firmware`` object is as shown below:
+
+.. parsed-literal::
+   :class: highlight
+
+   {
+   	"firmware": {
+		"fwVersion": "v1.0.3",
+		"fwLocation": {
+       			"protocol": "https:",
+			"host": "<hostname>",
+			"path": "<filename>.bin"
+		},
+		"fwFragmentSize" : 1800
```

Comment:
```suggestion
		"fwFragmentSize": 1800
```

Reply:
Not fixed.

Reply:
Should be okay now (also the other occurence)